lib/modules/manager/devbox/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { Category } from '../../../constants';
+import { NixhubDatasource } from '../../datasource/nixhub';
+import { NodeVersionDatasource } from '../../datasource/node-version';
+
+export { extractPackageFile } from './extract';
+export { updateArtifacts } from './artifacts';
+
+export const supportsLockFileMaintenance = false; // not yet
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)devbox\\.json$'],
+  commitMessageTopic: 'devbox/{{depName}}',
+  commitMessageExtra: 'to {{newValue}}',
+  enabled: true,
+};
+
+export const categories: Category[] = ['devbox', 'node'];
```

Comment:
```suggestion
export const categories: Category[] = ['node'];
```

Reply:
I think I will just remove these categories, I'm unsure if it is even required?